docs/docs/get-started.md

```diff
@@ -48,8 +48,10 @@ Recce expects two sets of dbt [artifacts](https://docs.getdbt.com/reference/arti
 - `target-base/` - dbt artifacts for to be used as the base for the comparison e.g. production
 - `target/` - dbt artifacts for your development branch
 
-#### Generate artifacts for the `base` environment
+#### Prepare artifacts for the `base` environment
+For most data warehouses, you can download the artifafcts from the codebase of your main branch. You don't need to re-run the whole production in your local. 
```

Comment:
> you can download the artifacts from the codebase of your main branch.
1. typo: artifacts -> artifacts
2. Do you think "you can download the artifacts **generated** from the codebase of your main branch."

Reply:
fixed. thanks!